resources/create-ingress.ts

```diff
@@ -53,9 +27,7 @@ export const createIngress = (values: CreateIngress) => {
             }
         },
         spec: {
-            ingressClassName: 'nginx',
-            tls: [tls],
```

Comment:
shouldn't we add these to patches (can't recall if we have these already added 🙏 )

Reply:
this is already present inside patches

Reply:
awesome, thanks! 🙏